src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/SupplierAssessmentService.kt

```diff
@@ -0,0 +1,30 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.service
+
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.AuthUser
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Referral
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.SupplierAssessment
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.repository.ReferralRepository
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.repository.SupplierAssessmentRepository
+import java.util.UUID
+import javax.transaction.Transactional
+
+@Service
+@Transactional
+class SupplierAssessmentService(
+  val supplierAssessmentRepository: SupplierAssessmentRepository,
+  val referralRepository: ReferralRepository,
+) {
+  fun createInitialAssessment(
+    referral: Referral,
+    createdByUser: AuthUser
```

Comment:
code smell here with this not being used ...

since this is created automatically does it make sense to have a createdBy field ?

we have a createdBy field on the appointments, so perhaps that's enough?